go.mod

```diff
@@ -502,3 +502,5 @@ replace (
 //
 // Then, run `make modtidy-all` in this repository.
 // This ensures that go.mod and go.sum are up-to-date for each go.mod file.
+
```

Comment:
Can you point this to your fork of components contrib, temporarily so tests can run?

Reply:
without waiting for the contrib PR to be merged ?

Reply:
Yes, we usually do this so all the tests can run in CI, otherwise the linter (`static checks`) will catch the `replace` change and won't allow the workflow to continue to tests (see here for example: https://github.com/dapr/dapr/actions/runs/13782463382). Speaking of which, pls do the change in the original import (line 14 in the go.mod), instead of the replace.
Once components contrib is merged you can update the pr here.

Reply:
I believe I need to modify my pseudo version to include the timestamp too

Reply:
@elena-kolevska should be good now or I will switch to the @branch reference